src/reflect/scala/reflect/api/Internals.scala

```diff
@@ -711,7 +711,8 @@ trait Internals { self: Universe =>
 
     trait SyntacticValDefExtractor {
       def apply(mods: Modifiers, name: TermName, tpt: Tree, rhs: Tree): ValDef
```

Comment:
I had to keep the old version of `apply` around as well because there was some kind of bootstrapping issue whereby a quasiquote in code inside `src/compiler` tries to call it. I'm not sure if there's a way to avoid this.

The error I got was:

```
[error] /Users/chris/code/scala/src/compiler/scala/tools/reflect/FastStringInterpolator.scala:93:7: not enough arguments for method apply: (mods: FastStringInterpolator.this.c.universe.Modifiers, name: FastStringInterpolator.this.c.universe.TermName, tpt: FastStringInterpolator.this.c.universe.Tree, rhs: FastStringInterpolator.this.c.universe.Tree, noWarnAttachment: Boolean)FastStringInterpolator.this.c.universe.ValDef in trait SyntacticValDefExtractor.
[error] Unspecified value parameter noWarnAttachment.
[error]       q"""{
[error]       ^
```